powerhome/playbook
# [PLAY-413] Date Picker Hide Input Icon broken

#### Screens

<img width="829" alt="Screen Shot 2023-02-13 at 3 09 31 PM" src="https://user-images.githubusercontent.com/73671109/218564109-bebc4bf6-9531-4677-b7c5-7991ef769f8c.png">

#### Breaking Changes

NO

#### Runway Ticket URL

[Runway Ticket](https://nitro.powerhrg.com/runway/backlog_items/PLAY-413)

#### How to test this

Use the hideIcon prop in the react version of the kit and check if Icon is hidden.

#### Checklist:
